loopflow-computation/src/main/java/com/farao_community/farao/loopflow_computation/LoopFlowComputationImpl.java

```diff
@@ -41,11 +41,13 @@
     protected ReferenceProgram referenceProgram;
     protected Network network;
     protected Map<EICode, LinearGlsk> glskMap;
+    protected String sensitivityProvider;
 
-    public LoopFlowComputationImpl(ZonalData<LinearGlsk> glsk, ReferenceProgram referenceProgram, Network network) {
+    public LoopFlowComputationImpl(ZonalData<LinearGlsk> glsk, ReferenceProgram referenceProgram, Network network, String sensitivityProvider) {
```

Comment:
i would put the new `sensitivityProider` argument in in the `calculateLoopFlows()` method instead, the method which also takes in argument a sensitivityAnalysisParameter.

Within the RAO, we only use the `buildLoopFlowsFromReferenceFlowAndPtdf()`, which rely on a previously calculated sensitivity result, and therefore which does not run internally a sensi

Reply:
makes sense, i'll change this